src/printer.rs

```diff
@@ -24,12 +31,27 @@ impl<'a> Printer<'a> {
             Colors::plain()
         };
 
-        Printer {
+        // Create the instance.
+        let mut instance = Printer {
             handle,
             colors,
             config,
+            panel_width: 0,
             line_changes: None,
-        }
+        };
+
+        // Generate the panel (gutter) width.
+        let decorations = instance.gen_decorations(0);
+        instance.panel_width = decorations.len()
+            + decorations.iter().fold(0, |a, x| a + x.size)
+            + if config.output_components.grid() {
```

Comment:
0 is added in both cases here? This can't be right?

Reply:
I used to count the grid line as part of the gutter's width, but ended up changing it later on to simplify calculations. My bad, it looks like I forgot to remove this leftover code.